### PR TITLE
New version

### PR DESCRIPTION
**Pull request by nhakmiller**
### Background

Prepping for version 0.5.3.

### Changes

* Bump versions

### Testing

* None

### CODE REVIEW

File: panther_analysis_tool/main.py [hunk 1]
```diff
@@ -1177,7 +1177,7 @@ def setup_parser() -> argparse.ArgumentParser:
         + "managing Panther policies and rules.",
         prog="panther_analysis_tool",
     )
-    parser.add_argument("--version", action="version", version="panther_analysis_tool 0.5.2")
+    parser.add_argument("--version", action="version", version="panther_analysis_tool 0.5.3")
```
Comment (wey-chiang) [inline panther_analysis_tool/main.py:1180]:
```suggestion
    parser.add_argument("--version", action="version", version="panther_analysis_tool 0.6.0")
```

File: setup.py [hunk 2]
```diff
@@ -20,14 +20,14 @@
...
-    version='0.5.2',
+    version='0.5.3',
     license='AGPL-3.0',
     description=
     'Panther command line interface for writing, testing, and packaging policies/rules.',
     author='Panther Labs Inc',
     author_email='pypi@runpanther.io',
     url='https://github.com/panther-labs/panther_analysis_tool',
-    download_url = 'https://github.com/panther-labs/panther_analysis_tool/archive/v0.5.2.tar.gz',
+    download_url = 'https://github.com/panther-labs/panther_analysis_tool/archive/v0.5.3.tar.gz',
```
Comment (wey-chiang) [inline setup.py:30]:
```suggestion
    download_url = 'https://github.com/panther-labs/panther_analysis_tool/archive/v0.6.0.tar.gz',
```

File: setup.py [hunk 3]
```diff
@@ -20,14 +20,14 @@
 setup(
     name='panther_analysis_tool',
     packages=['panther_analysis_tool'],
-    version='0.5.2',
+    version='0.5.3',
```
Comment (wey-chiang) [inline setup.py:23]:
```suggestion
    version='0.6.0',
```

**Review by wey-chiang — APPROVED** [review]
Given the amount of new features we're shipping, I think we should consider bumping the minor version

**Comment by nhakmiller** [comment]
> Given the amount of new features we're shipping, I think we should consider bumping the minor version

Sounds good to me.

**Pull request merged**